modules/services/nextcloud-client.nix

```
@@ -17,6 +17,13 @@ in {
         defaultText = literalExample "pkgs.nextcloud-client";
         description = "The package to use for the nextcloud client binary.";
       };
+
+      runInBackground = mkOption {
+        type = types.bool;
+        default = false;
+        defaultText = literalExample "false";
+        description = "Whether to run the nextcloud client in the background.";
```

Can you ever bring it out of the background, or does it stay in the background forever?

---

You can bring it out of the background by simply opening/running it.

---

would this be a more accurate description, then?
```suggestion
        description = "Whether to start the Nextcloud client in the background.";
```

---

Oh, much better.